hw/ip/entropy_src/doc/dv_plan/index.md

```diff
@@ -84,10 +84,10 @@ It creates the following analysis ports to retrieve the data monitored by corres
 ## Building and running tests
 We are using our in-house developed [regression tool]({{< relref "hw/dv/tools/README.md" >}}) for building and running our tests and regressions.
 Please take a look at the link for detailed information on the usage, capabilities, features and known issues.
-Here's how to run a basic sanity test:
+Here's how to run a basic smoke test:
```

Comment:
we think it would be good to remove the basic and just use `run a smoke test`

Reply:
Done